sdk/communication/communication-chat/src/chatClient.ts

```diff
@@ -42,12 +42,16 @@ import { getSignalingClient } from "./signaling/signalingClient";
 import { logger } from "./models/logger";
 import { tracingClient } from "./generated/src/tracing";
 
+declare interface InternalChatClientOptions extends ChatClientOptions {
```

Comment:
could we put it into the options.ts?

Reply:
No, because this should not be exported.